### PR TITLE
correcting the 116th question answer (suppose to be two correct options)

### PR DESCRIPTION
**Pull request by MogahidGaffar**
PR Type
 content changes (updating questions, answers, explanations or study resources)

What's new?
This PR corrects the answer for the 116th question, as it previously listed only one correct answer. 
The question now has two correct options.

Screenshots
Doc

### CODE REVIEW

File: content/questions/actions/question-116.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
 ---
-question: "When can you delete workflow runs?"
+question: "When can you delete workflow runs? (selct two)"
```
Comment (tfsjohan) [inline content/questions/actions/question-116.md:2]: There's spelling error in "selct" here.

**Review by tfsjohan — CHANGES_REQUESTED** [review]
Changes looks good, except for a small spelling error.

File: content/questions/actions/question-116.md (hunk 1 above)
Comment (MogahidGaffar) [reply content/questions/actions/question-116.md]: sorry , now typo corrected

**Review by tfsjohan — APPROVED** [review]

**Review by tfsjohan — APPROVED** [review]

**Review by tfsjohan — APPROVED** [review]

**Comment by tfsjohan** [comment]
Thanks for your help!

**Comment by MogahidGaffar** [comment]
my pleasure! 😄

**Comment by MogahidGaffar** [comment]
By the way, I have earned my GitHub action certification from Microsoft yesterday. Thank you, too. ! 💙

**Pull request merged**